cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricDatasetFactory.java

```diff
@@ -148,6 +150,9 @@ public static void setupDatasets(CConfiguration conf, DatasetFramework datasetFr
 
     // adding kafka consumer meta
     factory.createKafkaConsumerMeta();
+    DataMigration26 migration26 = new DataMigration26(conf, datasetFramework, factory);
```

Comment:
yeah, but this method is called by the existing upgrade tool, we will change where we call this based on the new upgrade tool.